### DISCUSSION TITLE
[@types/eslint@] is not in the npm registry.

### DISCUSSION THREAD

**Discussion by willocho** [DefinitelyTyped/DefinitelyTyped#61030] [Issues with a @types package]
Basically the title. npm run build is failing with this error "npm ERR! 404  '@types/eslint@*' is not in the npm registry.". It does look like there is nothing in the registry for @types/eslint@8.4.4

[a comment by an automated account is not shown]

**Reply by TheNickm2** [reply]
Also experiencing this issue. The README is also missing from the NPM registry @ 8.4.4

Version 8.4.3 working as expected (i.e. `npm i -D @types/eslint@8.4.3` works)

Opening an issue.

**Reply by willocho** [reply]
I didn't even realize this wasn't an issue. Thanks @TheNickm2
